### ISSUE TITLE
election sentiment rating

### ISSUE THREAD

**Issue by spm2164** [jsoma/playfair-projects#105]
If I could get the code the speaker from this Friday was using to analyze sentiment of tweets, I could run the transcripts of the conventions and maybe more stuff through it and see who is being more positive/negative and by how much.

- [x] My pitch issue has `[Pitch]` in the title
- [x] My pitch issue has `[Data request]` in the title if I need additional data
- [x] My pitch issue links to my data set
- [x] My pitch issue explains what I'd like to explore in the data set
- [x] My pitch issue includes multiple possible headlines
- [x] My pitch issue includes any links/images I'm using as inspiration
- [x] I have received two comments of peer feedback on my pitch issue

**Comment by rashidakamal** [comment]
https://github.com/williamsmj/sentiment

**Comment by palrogg** [comment]
Looks great! My only concern is about the size of data, if you gather many days, and the time the script could take to process it.

**Comment by sz2472** [comment]
I love this ambitious project! However, I am concerned about to narrow down your data set by filtering the time, the candidates and the convention dates.

**Comment by M0nica** [comment]
@palrogg @spm2164 

@jsoma had mentioned in slack: " I’d probably work on a subset on your computer, then when you get all of your code working spin up a digital ocean server with a large amount of RAM, upload your notebook to the server, download the dataset to the server, and run a notebook server on the machine"

Here are instructions if you're interested in going that route: https://www.digitalocean.com/community/tutorials/how-to-set-up-a-jupyter-notebook-to-run-ipython-on-ubuntu-16-04

**Comment by playfairbot** [comment]
Hi there, I'm the Playfair Bot! Would you mind posting the appropriate checklist in the main body of your issue? You might have posted it as the first comment, but it turns out it works **way better** in the actual body of the issue - just go up to the veeery top right and click the **pencil icon** to edit. You'll probably want to edit the comment to copy the checklist, then edit the original issue to paste it in. Thanks! :pray:

**Comment by spm2164** [comment]
So, I'm kind of just backfilling this right now since I didn't do any of this in the right order, but my original idea for a headline was something like "Who is more negative?" or something along those lines, with the feeling that the Trump speech would be like, way below zero and I could fill the space below the lines to the x-axis. Turns out, acceptance speeches are super positive overall. It wasn't until I had tracked down a decent amount of historical perspective with the other speeches that it became clear that my initial feeling, and the feeling of many who watched the speeches, was correct. D.Trump was historically negative in his speech.

No real dataset to link to, but my notebook will have the full text of all the speeches in there.
